clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -6454,8 +6454,7 @@ ConstantAddress CodeGenModule::GetAddrOfGlobalTemporary(
       !EvalResult.hasSideEffects())
     Value = &EvalResult.Val;
 
-  LangAS AddrSpace =
-      VD ? GetGlobalVarAddressSpace(VD) : MaterializedType.getAddressSpace();
```

Comment:
@yxsamliu, you may have added this back in cbf647cc3a712. Do you know when/if the AddrSpace should be set from MaterializedType since it seems VD is always non-null here.